### DISCUSSION TITLE
通过cloudflare解析后 添加自定义域名时出现queryCname ENODATA

### DISCUSSION THREAD

**Discussion by ZHEYESHIYU** [labring/sealos#3940] [Q&A]
cloudflare显示已代理成功
![image](https://github.com/labring/sealos/assets/142708826/a7cffc35-14a3-4bcc-a81d-ca5de1c021d6)
但是
![image](https://github.com/labring/sealos/assets/142708826/47b36352-c883-419b-b60f-af4ad89e899c)

之前同样的方法在zeabur和vercel上都可以实现
是我哪里操作出错了吗

**Comment by zhangchengmcbedrock** [comment]
把代理状态设置为仅 DNS

**Comment by 84519654** [comment]
我也是这个问题，大佬解决了吗


---
I have the same problem, has anyone solved it?
